client/debian_make_all.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -euo pipefail
+
+make clean
+sudo apt-get install -y g++ libgraphicsmagick++1-dev libavcodec-dev libavformat-dev libswscale-dev
```

Comment:
mmh, so if you want to include compiler related stuff you should choose the package `build-essential`. It not only includes the g++ but also make (because chances are, if g++ has to be installed, the system is totally bare and won't contain anything else needed for compilation).